### DISCUSSION TITLE
show IMDb rating of a movie.

### DISCUSSION THREAD

**Discussion by sanjublk** [Bugswriter/tuxi#168] [Ideas]
Can we get a feature where we get to see the IMDb rating of a movie along with its summary.
i look for the movie ratings whenever my friends suggests a movie.it would be great if i can do that with Tuxi<3

**Comment by bugswriter** [comment]
yes not from IMBD ofc. Tomorrow I will work on this.

**Reply by BeyondMagic** [reply]
alright, @Bugswriter I think I did it

@sanjublk 

![image](https://user-images.githubusercontent.com/51831435/110057862-1cd48200-7d40-11eb-9e79-ab24e79e62c7.png)

**Reply by bugswriter** [reply]
that's what we need PR for purert-version possible?

**Reply by BeyondMagic** [reply]
I think it still has a few problems, but I'm working on it.

This scraping happens at `kno_right`, but it scrapes a lot of other things, too, besides movies, so I need to see how can I scrape this only for movies!
